typescript/packages/subsurface-viewer/src/storybook/components/settings/LayerSettingsButton.stories.tsx

```diff
@@ -6,29 +6,31 @@ export default {
     title: "SubsurfaceViewer/Components/Settings/LayerSettingsButton",
 };
 
-const Template = (args) => {
-    return <LayerSettingsButton layer={args.layer} />;
-};
+// Note: Disabled for now will be fixed later. Missing redux context.
 
-export const DrawingSettings = Template.bind({});
-DrawingSettings.args = {
-    layer: {
-        name: "Drawing",
-        "@@type": "DrawingLayer",
-        id: "drawing-layer",
-        visible: true,
-        mode: "drawLineString",
-    },
-};
+// const Template = (args) => {
```

Comment:
Suggest removing instead of commenting out. The code is still in the git history.